### ISSUE TITLE
Cannot resolve dependency '@fortawesome/fontawesome-free-webfonts/fontawesome.css'

### ISSUE THREAD

**Issue by mwinkler** [parcel-bundler/parcel#626]
**Choose one:** is this a 🐛 bug report

I try to import the new font awesome 5 npm package, and get this error:

```js
🚨  C:\_Dev\My\Github\Playground\Parcel\src\index.js:7:7: Cannot resolve dependency '@fortawesome/fontawesome-free-webfonts/fontawesome.css'
>  7 | import  '@fortawesome/fontawesome-free-webfonts/fontawesome.css'
     |          ^
```
[FontAwesome - JS Component Packages](https://fontawesome.com/how-to-use/js-component-packages)

### 🎛 Configuration (.babelrc, package.json, cli command)

Just package.json

```js
{
  "name": "Parcel",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "parcel ./src/index.html"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "parcel-bundler": "^1.4.1"
  },
  "dependencies": {
    "@fortawesome/fontawesome-free-webfonts": "^1.0.2"
  }
}

```

cli cmd
`npm start`

### 🤔 Expected Behavior

Import should work

### 😯 Current Behavior

See error above

### 💻 Code Sample

index.html
```html
<!DOCTYPE html>
<html lang="en">
<body>
    <h1>Test <i class="fas fa-camera-retro"></i></h1>
    <script src="./index.js"></script>
</body>
</html>
```

index.js
```js
import '@fortawesome/fontawesome-free-webfonts/fontawesome.css'

console.log('Hello');
```

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           |1.4.1|
| Node             |8.9.1|
| npm/Yarn         |5.5.1|
| Operating System |Win10-64bit (1607)

**Comment by osdevisnot** [comment]
It seems there are few things that are incorrect here.

* you are importing `import '@fortawesome/fontawesome-free-webfonts/fontawesome.css';`, but this file does not exist on that path, please import this instead `import '@fortawesome/fontawesome-free-webfonts/css/fontawesome.css';` (notice `/css/` before the filename). 
* per the package definition of font awesome the `main` entry in `package.json` points to above file already, so you should be able to drop filename altogether and still be able to load fontawesome correctly. eg: `import '@fortawesome/fontawesome-free-webfonts';`

* Also, the new font awesome has 3 font face definitions, namely: regular (`far` classname), solid (`fas` classname) and brand (`fab` classname). To use either one of these you have to import the individual face definition css as well. For example, for solid fonts that you show in your example `fas fa-camera-retro`, you should import `import '@fortawesome/fontawesome-free-webfonts/css/fa-solid.css';`

> TLDR; change your index.js to:
```
import '@fortawesome/fontawesome-free-webfonts';
import '@fortawesome/fontawesome-free-webfonts/css/fa-solid.css';

console.log('Hello');
```

**Comment by mwinkler** [comment]
Thank you!
